multinet/db.py

```diff
@@ -307,13 +308,13 @@ def delete_graph(workspace: str, graph: str) -> str:
     return graph
 
 
-def graph_node_tables(workspace: str, graph: str) -> List[StandardCollection]:
+def graph_node_tables(workspace: str, graph: str) -> List[str]:
     """Return the node tables associated with a graph."""
     g = get_graph_collection(workspace, graph)
     return g.vertex_collections()
 
 
-def graph_edge_table(workspace: str, graph: str) -> Optional[StandardCollection]:
+def graph_edge_table(workspace: str, graph: str) -> StandardCollection:
```

Comment:
I'm a little surprised this passes typechecking, since the function still looks like it can return `None`. My feeling is this function really should be `Optional[StandardCollection]` and we need to figure out the type narrowing stuff at the sites where this function is called, or we need to not return `None` from here and instead raise an appropriate 500 error if the condition arises.

The latter might be the easiest thing to do if the actual risk of the error condition arising is small (which my intuition says it is).

Reply:
I agree. The only time this should happen is when something is seriously wrong with our application (i.e. a graph shouldn't exist without an edge definition) so I think we should raise an error here. Maybe `500 CorruptedGraph` or something?

Reply:
I think just the generic 500 error ("Internal Server Error") is fine. This would be picked up by sentry and we could investigate. It's not a common enough error that we should name it.

Reply:
Sounds good